lib/process-services-cloud/src/lib/form/components/widgets/attach-file/file-properties-table-cloud.component.html

```diff
@@ -40,7 +40,7 @@
             <th mat-header-cell *matHeaderCellDef></th>
             <td mat-cell *matCellDef="let element">
                 <button id="{{'file-'+element?.id+'-option-menu'}}" mat-icon-button [matMenuTriggerFor]="fileActionMenu"
-                    *ngIf="!!element.content?.mimeType">
+                    *ngIf="!!element.content?.mimeType" [title]="'ADF_PROCESS_LIST.DETAILS.BUTTON.FILE_OPTION_MENU' | translate">
```

Comment:
Should we really use title attribute or rather aria-label? Using title may affect on design because it adds tooltip but aria-label is just for reading by screen readers. @MichalKinas what do you think?

Reply:
@AleksanderSklorz I agree, aria-label should be the default solution

Reply:
should I change to aria label, what do you say @MichalKinas @AleksanderSklorz

Reply:
@jatin2008 yes, if you don't specifically want to add the title with tooltip then aria-label should be used

Reply:
I agree too

Reply:
noted and addressed